src/clockcache.h

```diff
@@ -116,14 +116,14 @@ struct clockcache {
 
    uint32              *lookup;
    clockcache_entry    *entry;
-   buffer_handle       *bh;   // actual memory for pages
+   buffer_handle        bh;   // actual memory for pages
    char                *data; // convenience pointer for bh
    platform_log_handle *logfile;
    platform_heap_handle heap_handle;
```

Comment:
The `heap_handle` field appears to be dead after these changes.

Reply:
Yes; you are right. Took it out.